src/sonic_ax_impl/mibs/ieee802_1ab.py

```diff
@@ -317,6 +317,9 @@ def reinit_data(self):
                 if '.' in mgmt_ip:
                     mgmt_ip_sub_oid = (addr_subtype_sub_oid, *[int(i) for i in mgmt_ip.split('.')])
                     break
+            else:
```

Comment:
else block is not intended to match the right if block.

Reply:
This is for-else structure, which is a critical change in this PR. If there is no IPv4 address matched in the for loop, we need to return soon. Otherwise future statement will fail since mgmt_ip_sub_oid==None

---
In reply to: [503552324](https://github.com/Azure/sonic-snmpagent/pull/164#discussion_r503552324) [](ancestors = 503552324)

Reply:
got it, thank you.